yb-voyager/cmd/exportDataStatusCommand.go

```diff
@@ -178,11 +181,14 @@ func runExportDataStatusCmd() ([]*exportTableMigStatusOutputRow, error) {
 		}
 		displayTableName := finalFullTableName.ForMinOutput()
 		partitions := leafPartitions[finalFullTableName.ForOutput()]
-		if source.DBType == POSTGRESQL && partitions != nil {
+		if source.DBType == POSTGRESQL && partitions != nil && msr.IsExportTableListSet {
```

Comment:
Can you add a comment here explaining why we want to change the display name only if table-list is specified?

Reply:
Yeah, I will add it. Reason: This is just for display in case the user has provided a table list and in case he has subset of leaf tables so let them know that we are only exporting those not all.